### DISCUSSION TITLE
Command to create symlink for runtime on Windows works in Windows Powershell but not in Powershell

### DISCUSSION THREAD

**Discussion by bluthej** [helix-editor/helix#10196] [General]
The command provided in the [Installation guide](https://docs.helix-editor.com/install.html#configuring-helixs-runtime-files) to create a symlink for the runtime folder works in "Windows Powershell" (provided by default with Windows), but not in "[Powershell](https://github.com/PowerShell/PowerShell)" (the cross-platform application which started as a fork of Windows Powershell).

I am running Powershell version 7.6.3, and I think terminal-heavy Windows users will likely prefer Powershell over Windows Powershell, so it would be nice to have a command that works in both applications like:

```powershell
New-Item -ItemType Junction -Target "$PWD\runtime" -Path "$Env:AppData\helix\runtime"
```

**Comment by AucaCoyan** [comment]
Today I had this issue, and I realized it wasn't `%APPDATA%` that because I installed helix with scoop
So, the directory is `~/scoop/persist/helix/config.toml`

You can find your config with the helix command:
```
:config-open
```
